### PR TITLE
Add the SvgMessage and SvgMessageRound icons

### PR DESCRIPTION
**Pull request by ob6160**
## What are you changing?

- Add the Message and MessageRound icons to `source-react-components` and export them

### Screenshot (changes indic. with red box)
Screenshot 2023-02-23 at 17 28 06

### CODE REVIEW

**Review by joecowton1 — APPROVED** [review]

File: libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		height={undefined}
+		viewBox="-3 -3 30 30"
+		xmlns="http://www.w3.org/2000/svg"
+		focusable={false}
+		aria-hidden={true}
+	>
+		<path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="M18.75 3v2.571H3V3h15.75Zm-5.464 5.143v2.571H3V8.143h10.286ZM21 15.857v-2.571H3v2.571h18Zm-4.5 2.572v2.57H3v-2.57h13.5Z"
```
Comment (mxdvl) [inline libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx:20]: I am baffled by the lack of concern for the pixel grid in our designs…

**Review by mxdvl — COMMENTED** [review]
jUSt_a+houg4+=?!

File: libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx (hunk 1 above)
Comment (akemitakagi) [reply libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx:20]: @mxdvl Let's chat.

**Comment by akemitakagi** [comment]
Thank you, Ollie. 👍

File: libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx (hunk 1 above)
Comment (mxdvl) [reply libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx:20]: 😁

**Review by akemitakagi — APPROVED** [review]

**Comment by akemitakagi** [comment]
@ob6160 I updated the message icon in Figma.
Would you mind pulling the message icon again, please?

File: libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx (hunk 1 above)
Comment (mxdvl) [reply libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx:20]: We discussed this offline and this is one of these icons that have been transferred from Illustrator to Figma with low fidelity. Fixing in the authoring software is not easy, hence these unsnapped values.

**Comment by ob6160** [comment]
> @ob6160 I updated the message icon in Figma. Would you mind pulling the message icon again, please?

great - thanks for letting me know, I'll pull the updates on Monday as I'm off work today.

Thanks for bringing up the Grid alignment too @mxdvl, I learnt something interesting there that I wouldn't have noticed before :)

File: libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const Svg = ({ size }: IconProps): EmotionJSX.Element => (
 		<path
 			fillRule="evenodd"
 			clipRule="evenodd"
-			d="M18.75 3v2.571H3V3h15.75Zm-5.464 5.143v2.571H3V8.143h10.286ZM21 15.857v-2.571H3v2.571h18Zm-4.5 2.572v2.57H3v-2.57h13.5Z"
+			d="M19 3.5V6H3V3.5h16Zm-5.5 5V11H3V8.5h10.5ZM21 16v-2.5H3V16h18Zm-4.5 2.5V21H3v-2.5h13.5Z"
```
Comment (mxdvl) [inline libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx:20]: 👏  so neat 📐

**Pull request merged**